<launch>
<node pkg="laser_filters" type="scan_to_scan_filter_chain" output="screen" name="shadow_laser_filter">
      <remap from="scan" to="base_scan" />
      <remap from="scan_filtered" to="shadow_filtered" />
      <rosparam command="load" file="$(find jsk_2013_04_pr2_610)/launch/shadow_filter_example.yaml" />
</node>
<node pkg="laser_filters" type="scan_to_scan_filter_chain" output="screen" name="footobject_laser_filter">
      <remap from="scan" to="shadow_filtered" />
      <remap from="scan_filtered" to="scan_filtered_foot" />
      <rosparam command="load" file="$(find laser_filters_jsk_patch)/launch/footobject_filter_example.yaml" />
</node>
</launch>
